src/CqlBuilderPanel/ValueSets/Search/Search.tsx

```diff
@@ -149,6 +163,16 @@ export default function Search(props: SearchProps) {
                       prefix="Search"
                       label={SEARCH_MAP[value]}
                       trimField={trimField}
+                      placeHolder={
+                        value == "version" && formik.values.url.length === 0
+                          ? "OID/URL must be entered first"
+                          : ""
+                      }
+                      disabled={
+                        value == "version"
+                          ? formik.values.url.length === 0
+                          : false
+                      }
```

Comment:
In general you do not want to use == in javascript as it could cause unforseen issues since it's not strict equality. 

this check
formik.values.url.length === 0
could be the same as !formik.values.url to account for edge case when value is undefined and not a string. 
Doesn't seem to hit this, but it's a safe way to do it other places